tests/framework/builder.py

```diff
@@ -55,17 +55,21 @@ def build(self,
               config: Artifact,
               net_ifaces=None,
               enable_diff_snapshots=False,
-              cpu_template=None):
+              cpu_template=None,
+              use_ramdisk=False):
         """Build a fresh microvm."""
         vm = init_microvm(self.root_path, self.bin_cloner_path,
```

Comment:
To avoid having to create a ramdisk for every single jailer instance (and since most tests don't need it), I think we can pass the `use_ramdisk` parameter down to the `init_microvm` function and then to the `Microvm` instance as a constructor parameter.